docs/_static/auth/iam.yaml

```diff
@@ -753,8 +729,133 @@ paths:
           $ref: '../common/error_response.yaml'
 components:
   schemas:
+    cnf:
+      description: The 'confirmation' claim is used in JWTs to proof the possession of a key.
+      required:
+        - jkt
+      properties:
+        jkt:
+          type: string
+          description: JWK thumbprint
     DIDDocument:
       $ref: '../common/ssi_types.yaml#/components/schemas/DIDDocument'
+    DPoPRequest:
+      type: object
+      required:
+        - htm
+        - token
+        - htu
+      properties:
+        htm:
+          type: string
+          description: The HTTP method for which the DPoP proof is requested.
+          example: "POST"
+        token:
+          type: string
+          description: The access token for which the DPoP proof is requested.
+          example: "eyJhbGciOi"
+        htu:
+          type: string
+          description: The URL for which the DPoP proof is requested. Query params and fragments are ignored during validation.
+          example: "https://example.com/resource"
+    DPoPResponse:
+      type: object
+      required:
+        - dpop
+      properties:
+        dpop:
+          type: string
+          description: The DPoP proof as specified by https://datatracker.ietf.org/doc/html/rfc9449 for resource requests
+    DPoPValidateRequest:
+      type: object
+      required:
+        - dpop_proof
+        - method
+        - thumbprint
+        - token
+        - url
+      properties:
+        dpop_proof:
+          type: string
+          description: The DPoP Proof as specified by https://datatracker.ietf.org/doc/html/rfc9449 for resource requests
```

Comment:
```suggestion
          description: The value of the DPoP Header provided in the resource requests, https://datatracker.ietf.org/doc/html/rfc9449
```

Reply:
conflicting reviews, DPoP proof vs DPoP header value. Now it's DPoP proof everywhere.